src/main/java/io/codelex/flightplanner/config/WebSecurity.java

```diff
@@ -0,0 +1,22 @@
+package io.codelex.flightplanner.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.web.SecurityFilterChain;
+
+@EnableWebSecurity
+public class WebSecurity {
+    @Bean
+    public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
```

Comment:
Nu šis ir tā slideni, bet nu var atstāt... :D Laikam sanāk, ka es pateicis, ka viss zem admin-api  ir ar paroli, bet viss pārējais ir bez. Tas šajā gadījumā strādā pilnībā korekit, bet ir jauzmanās - ja pievieno jaunu kaut kādu path - tas automātiski būs bez paroles - labāk ir konfigurēt otrādāk - visam parole un tikai atsevišķām lietām nav. Bet nu - vari atstāt arī šādi.